spacy_loggers/chain.py

```diff
@@ -0,0 +1,52 @@
+"""
+A utility logger that allows multiple loggers to be daisy-chained.
+"""
+from typing import Dict, Any, Optional, IO
+import sys
+
+from spacy import Language
+from .util import LoggerT
+
+
+def chain_logger_v1(
+    logger1: LoggerT,
+    logger2: LoggerT,
```

Comment:
I think they should all be optional. You could imagine someone wanting to keep the rest of their config the same while swapping out various loggers, so I think it's more flexible to keep them all optional. It wouldn't be obvious from looking at the config that particular ones were required, either.

Reply:
Fair point; will change.